Containers/talk/supervisord.conf

```diff
@@ -13,14 +13,14 @@ stdout_logfile=/dev/stdout
 stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
-command=turnserver
+command=turnserver -c /etc/turnserver.conf
 
 [program:nats-server]
 stdout_logfile=/dev/stdout
 stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
-command=nats-server -c /etc/nats/nats.conf
+command=nats-server -c /etc/nats.conf
 
 [program:janus]
```

Comment:
we should check if we can somehow disable these plugins:
janus.transport.rabbitmq
janus.transport.pfunix

Reply:
but what is the context of this with supervisor?

Reply:
not the correct place indeed, possibly only if we can disable it with a command line option... will need to look into janus docs how to disable these plugins...

Reply:
both have set `enabled = false` in their config file